Fundamentals-2-Ind.md

```diff
@@ -24,7 +24,7 @@ Such strategy or policy might also be needed on product or product area level.
 
 ![Infrastructure](./img/Infrastructure.png)
 
-_Image by [Randall Munroe](https://xkcd.com/2347/)_
+_Image by [Randall Munroe](https://xkcd.com/2347/){:target="_blank"}_
 
```

Comment:
Very good keeping the original page open while allowing readers to explore linked page outside.